templates/jest.config.json

```diff
@@ -5,7 +5,10 @@
   },
   "moduleNameMapper": {
     "^\\$lib(.*)$": "<rootDir>/src/lib$1",
-    "^\\$app(.*)$": ["<rootDir>/.svelte-kit/dev/runtime/app$1", "<rootDir>/.svelte-kit/build/runtime/app$1"]
+    "^\\$app(.*)$": [
+      "<rootDir>/.svelte-kit/dev/runtime/app$1",
```

Comment:
The build folder was generated after `npm run build` and I am not sure how to generate the dev one but this is my folder content

<img width="423" alt="image" src="https://user-images.githubusercontent.com/23623792/168991539-5407db62-74fa-467b-ac70-6a53e81bd3d5.png">

I updated these lines to just `"<rootDir>/.svelte-kit//dev/runtime/app$1"` and it was working for me, let me know what you think @rossyman